### ISSUE TITLE
Add a Vintage graphics mode.

### ISSUE THREAD

**Issue by Dan-Linden** [secondlife/viewer#3135]
### Test plan
https://lindenlab.atlassian.net/wiki/spaces/QA/pages/3407052804/Vintage+Graphics+Mode+and+GL3+Fallback+Mode+test

### Environment

older systems

### Description

Add a mode that looks like FS 6.

### Reproduction steps

1. Start the viewer on older hardware ___
2. Verify ____ settings are enabled.
3. Verify it looks like _____.

**Comment by DmitrySProductengine** [comment]
Failed QA.
Verified on the Second Life Release 7.1.11.12041172537 (64bit) on Win10 with Intel HD Graphics 4000 video card in the scope of [IQA #401](https://github.com/secondlife/iqa/issues/401).
Enabling/disabling the new feature works fine. 
However, the default settings for old video cards behave differently from what is specified in the test plan.
I filed a ticket for this behavior: https://github.com/secondlife/viewer/issues/3167

The behavior of various video cards when enabling this feature will be tested in a separate task https://github.com/secondlife/iqa/issues/403.
